src/main/java/core/basesyntax/Application.java

```diff
@@ -3,5 +3,9 @@
 public class Application {
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+        String first = new Lottery().getRandomBall().toString();
+        String second = new Lottery().getRandomBall().toString();
+        String third = new Lottery().getRandomBall().toString();
+        System.out.println(first + second + third);
```

Comment:
```suggestion
        Lottery lottery = new Lottery();
        String first = lottery.getRandomBall().toString();
        String second = lottery.getRandomBall().toString();
        String third = lottery.getRandomBall().toString();
        System.out.println(first + second + third);
```
there is no need to create a lottery object for each call, once is enough

Reply:
and do it in loop, please